subsys/bluetooth/controller/hci/hci.c

```diff
@@ -74,6 +75,9 @@ static u64_t event_mask = DEFAULT_EVENT_MASK;
 static u64_t event_mask_page_2 = DEFAULT_EVENT_MASK_PAGE_2;
 static u64_t le_event_mask = DEFAULT_LE_EVENT_MASK;
 
+static void le_conn_complete(u8_t status, struct radio_le_conn_cmplt *radio_cc,
+			     u16_t handle, struct net_buf *buf);
```

Comment:
I'll assume this forward declaration was inevitable and it would have been impractical to move the function higher up instead?

Reply:
Not only impractical but it would also mess with the natural HCI order everything is defined in.